ArcGISRuntimeSDKQt_SampleViewer/mac/Info.plist

```diff
@@ -3,13 +3,13 @@
 <plist version="1.0">
 <dict>
     <key>CFBundleExecutable</key>
-    <string>ArcGISQt_CppSamples</string>
+    <string>ArcGISQt_Samples</string>
     <key>CFBundleGetInfoString</key>
     <string>Created by Qt/QMake</string>
     <key>CFBundleIconFile</key>
     <string>ArcGISRuntimeSDKQtSamples.icns</string>
     <key>CFBundleIdentifier</key>
-    <string>com.esri.ArcGISQt-CppSamples</string>
+    <string>com.esri.ArcGISQt-Samples</string>
```

Comment:
these changes could possibly have implications on the signing

Reply:
Yes, we need to be careful. It should work, but seemingly benign changes like this can cause issues. It's the right change, so let's try it out at least.

Reply:
If we prefer to make this change now, Do we want to keep with the name of the project instead? `com.esri.ArcGISQt-Samples` -> `com.esri.SampleViewer`?